src/functions.php

```diff
@@ -91,10 +92,16 @@ function configureScope(callable $callback): void
  * is automatically removed once the operation finishes or throws.
  *
  * @param callable $callback The callback to be executed
+ *
+ * @return mixed|void The callback's return value, upon successful execution
+ *
+ * @phpstan-template T
```

Comment:
To keep consistency with what it was done in the rest of the codebase, can you please put a blank line between each `@phpstan-*` annotation? Also, please use `@psalm-*` annotations

Reply:
Done